tests/acceptance/features/webUIFirstrunwizard/reset_all.feature

```diff
@@ -0,0 +1,26 @@
+@webUI @insulated @disablePreviews
+Feature:  first login wizard
+  As a user
+  I want to see basic features of ownCloud when I first log in to owncloud
+  So that I can get basic overview about ownCloud
```

Comment:
As an administrator
I want to be able to reset the first run wizard display status for all users
So that I can force users to be shown the first run wizard popup again